receiver/redisreceiver/metadata.yaml

```diff
@@ -112,21 +113,21 @@ metrics:
   redis.clients.max_input_buffer:
     enabled: true
     description: Biggest input buffer among current client connections
-    unit: ""
+    unit: "By"
     gauge:
       value_type: int
 
   redis.clients.max_output_buffer:
     enabled: true
     description: Longest output list among current client connections
-    unit: ""
+    unit: "By"
```

Comment:
Technically these two are breaking changes - will you please submit these as a separate PR?

Reply:
Thanks for the review! Removed from this PR and opened a separate PR with a `changelog` here: #23454.